### DISCUSSION TITLE
https://gay-hotfile.errio.net down - this is HFGPM, correct?

### DISCUSSION THREAD

**Discussion by jfreiman** [CodyBerenson/PGMA-Modernized#322] [General]
Just letting you know that this website has been down for about 4-5 days now...
[](https://gay-hotfile.errio.net/)

Does anyone know anything about this website?  I can't seem to find anything online about it's status, what happened, or if they are coming back. ;(

**Comment by JPH71** [comment]
Yes it is..
It's working as I used it today..

**Reply by jfreiman** [reply]
<img width="641" alt="image" src="https://github.com/user-attachments/assets/44b3af46-1494-4da8-9305-4e11c260e66a">
I just noticed!! It was still down earlier today and now it's up!

Thanks for checking!

**Reply by jfreiman** [reply]
It seems to be accessible and then it isn't.  Probably a lot of users trying to access the content.
I'm just very happy that they are still online.  It makes one very thankful there are sites like this available.

**Comment by JPH71** [comment]
You are very welcome xxx

**Comment by jfreiman** [comment]
![image](https://github.com/user-attachments/assets/1c12e7b2-ecef-45b4-b9de-abff852bffa1)
~8:30pm PST.....  🤞

**Reply by Jubaxrd** [reply]
Guys please invite to your server my user name is :Jubax Rd
